contrib/macdeploy/extract-osx-sdk.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+set -e
+
+7z x Xcode_7.3.1.dmg 5.hfs
+SDKNAME="MacOSX10.11.sdk"
+SDKDIRINODE=154283
+fls 5.hfs -rpF ${SDKDIRINODE} |
+ while read type inode filename; do
+	inode="${inode::-1}"
+	if [ "${filename:0:14}" = "usr/share/man/" ]; then
+		continue
+	fi
+	filename="${SDKNAME}/$filename"
+	echo "Extracting $filename ..."
+	mkdir -p "$(dirname "$filename")"
+	if [ "$type" = "l/l" ]; then
+		ln -s "$(icat 5.hfs $inode)" "$filename"
+	else
+		icat 5.hfs $inode >"$filename"
+	fi
+done
+echo "Building ${SDKNAME}.tar.gz ..."
+tar -c MacOSX10.11.sdk | gzip -9 > "${SDKNAME}.tar.gz"
```

Comment:
Using the tricks here: https://github.com/bitcoin/bitcoin/blob/v0.13.0/contrib/gitian-descriptors/gitian-linux.yml#L144
you could make the tar and tgz deterministic. That may be useful for people to compare their results?